hoot-core-test/src/test/cpp/hoot/core/io/ServiceOsmApiDbSqlChangesetFileWriterTest.cpp

```diff
@@ -87,24 +83,17 @@ class ServiceOsmApiDbSqlChangesetFileWriterTest : public HootTestFixture
     //clear out the db so we get consistent next id results
     database.open(ServicesDbTestUtils::getOsmApiDbUrl());
     database.deleteData();
-    const QString scriptDir = "test-files/servicesdb";
-    ApiDb::execSqlFile(ServicesDbTestUtils::getOsmApiDbUrl().toString(), scriptDir + "/users.sql");
+    ApiDb::execSqlFile(ServicesDbTestUtils::getOsmApiDbUrl().toString(), _scriptDir + "users.sql");
 
     OsmApiDbSqlChangesetFileWriter writer(ServicesDbTestUtils::getOsmApiDbUrl());
     //  Set the changeset max size to 5 (half of the changes) for this test only
     Settings testSettings = conf();
     testSettings.set("changeset.max.size", "5");
     writer.setConfiguration(testSettings);
     writer.setChangesetUserId(1);
-    writer
-      .write(
-        "test-output/io/ServiceOsmApiDbSqlChangesetFileWriterTest/changeset.split.osc.sql",
-        changesetProvider);
-    HOOT_STR_EQUALS(
-      TestUtils::readFile(
-        "test-files/io/ServiceOsmApiDbSqlChangesetFileWriterTest/changeset.split.osc.sql"),
-      TestUtils::readFile(
-        "test-output/io/ServiceOsmApiDbSqlChangesetFileWriterTest/changeset.split.osc.sql"));
+    writer.write(_outputPath + "changeset.split.osc.sql", changesetProvider);
+    HOOT_FILE_EQUALS(_inputPath + "changeset.split.osc.sql",
+                    _outputPath + "changeset.split.osc.sql");
```

Comment:
There were a few instances of using `HOOT_STR_EQUALS` to compare the contents of two files.  It is easier to read using `HOOT_FILE_EQUALS`.